ovos_tts_plugin_server/__init__.py

```diff
@@ -27,9 +27,12 @@ def __init__(self, *args, **kwargs):
             )
 
     @property
-    def host(self) -> Optional[str]:
+    def host(self) -> Optional[list]:
```

Comment:
This property no longer returns None for any reason, so it's simply `def host(self) -> list`. We'll want to make sure any code that was handling host being None is adjusted to account for that change

Reply:
@builderjer This is still valid, the return isn't Optional anymore, you always return List[str] now

Reply:
I'm not sure I am following.  Using `self.config.get("host")` if it fails will return None and therefore skip the `if` and return None, so still optional.  It does not need to be there.

Reply:
Ooo, good point, but that's actually worse...now you have the possibility of a list of strings, or a list of NoneType, and that list of NoneType is going to cause you problems later. Either way you will always return a list from this property.

hosts will be a string, a list of strings, or None. If it's a string or None, you force it into a list, giving you a list with a string or a list with None. Then you return hosts.

hosts will never return as None

Reply:
I still don't see how this returns a list if `self.config.get("host")` returns None it skips the whole converting to a list.

Reply:
You're right - I popped this open in my IDE and did some debugging, as well as let it infer what the return would be.